functions/truncate/1.0/index.js

```diff
@@ -0,0 +1,15 @@
+const truncate = async ({ value, size, trailing }) => {
+  const trailingText = trailing || '...';
+  let truncatedValue = value;
+  if (value.length > size) {
+    if (size <= 3) {
+      truncatedValue = value.slice(0, size - 3) + trailingText;
+    } else {
+      truncatedValue = value.slice(0, size) + trailingText;
+    }
+  }
+
+  return { result: truncatedValue };
+};
+
+export default truncate;
```

Comment:
Perhaps a better solution is to use Lodash truncate: https://lodash.com/docs/4.17.15#truncate